code/game/objects/structures/orbital_cannon.dm

```diff
@@ -181,28 +192,15 @@
 
 	ob_cannon_busy = TRUE
 
-	var/inaccurate_fuel = 0
-
-	switch(tray.warhead.warhead_kind)
-		if("explosive")
-			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[1] - tray.fuel_amt)
-		if("incendiary")
-			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[2] - tray.fuel_amt)
-		if("cluster")
-			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[3] - tray.fuel_amt)
-		if("plasma")
-			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[4] - tray.fuel_amt)
-
-	var/turf/target = locate(T.x + inaccurate_fuel * pick(-1,1),T.y + inaccurate_fuel * pick(-1,1),T.z)
-
+	var/turf/target = locate(T.x,T.y,T.z)
 	playsound_z_humans(target.z, 'sound/effects/OB_warning_announce.ogg', 100) //for marines on ground
 	playsound(target, 'sound/effects/OB_warning_announce_novoiceover.ogg', 125, FALSE, 30, 10) //VOX-less version for xenomorphs
 	playsound_z(z, 'sound/effects/OB_warning_announce.ogg', 100) //for the ship
 
 	var/impact_time = 10 SECONDS + (WARHEAD_FLY_TIME * (GLOB.current_orbit/3))
 
 	addtimer(CALLBACK(src, /obj/structure/orbital_cannon/proc/handle_ob_firing_effects, target), impact_time - (0.5 SECONDS))
-	var/impact_timerid = addtimer(CALLBACK(src, /obj/structure/orbital_cannon.proc/impact_callback, target, inaccurate_fuel), impact_time, TIMER_STOPPABLE)
+	var/impact_timerid = addtimer(CALLBACK(src, /obj/structure/orbital_cannon.proc/impact_callback, target), impact_time, TIMER_STOPPABLE)
```

Comment:
ok but innacurate fuel was used in the function, find out what it does

Reply:
I believe that the only function it served would be to increase the time between shooting and impact if it was not loaded with the required amount of solid fuel.

Since it is no longer possible to fire it without the required amount of solid fuel, it doesn't serve any purpose and has such been deleted for this function.

Reply:
no, you just deleted it from the argument. It's still in the function

Reply:
Ah i believe i missed them due to only deleting the ones that caused errors, so i missed the ones that didn't, i have now since deleted all references of `inaccurate_fuel` from the code.